### PR TITLE
Added search, labels and delete row styling

### PR DESCRIPTION
**Pull request by RicardasRimsevicius**

### CODE REVIEW

**Comment by ktf** [comment]
Please squash the commits.

File: stats/pending-pr-chart.js [hunk 1]
```diff
@@ -201,29 +201,32 @@ function writeMainTable() {
...
-            if (labelLetter == "P") {
-                categoriesPending += "<span class='label label-warning'>" + categories[o] + "</span>" + " ";
-            } else if (labelLetter == "A") {
-                categoriesPending += "<span class='label label-success'>" + categories[o] + "</span>" + " ";
-            } else if (labelLetter == "R") {
-                categoriesPending += "<span class='label label-danger'>" + categories[o] + "</span>" + " ";
-            } else if (labelLetter == "S") {
-                categoriesPending += "<span class='label label-warning'>" + categories[o] + "</span>" + " ";
-            }
+			if (o != 15) {
```
Comment (ktf) [inline stats/pending-pr-chart.js]: Use 2 spaces to indent.

File: stats/pending-pr-chart.js [hunk 2]
```diff
@@ -388,22 +410,24 @@ function changeCategoryData(category) {
...
-                if (labelLetter == "P") {
-                    categoriesPending += "<span class='label label-warning'>" + categories[o] + "</span>" + " ";
-                } else if (labelLetter == "A") {
-                    categoriesPending += "<span class='label label-success'>" + categories[o] + "</span>" + " ";
-                } else if (labelLetter == "R") {
-                    categoriesPending += "<span class='label label-danger'>" + categories[o] + "</span>" + " ";
-                } else if (labelLetter == "S") {
-                    categoriesPending += "<span class='label label-warning'>" + categories[o] + "</span>" + " ";
-                }
+				if (o != 15) {
```
Comment (ktf) [inline stats/pending-pr-chart.js]: Also here. Do not use a bunch of if statements, use a switch or a lookup table.

**Comment by RicardasRimsevicius** [comment]
Done

**Pull request merged**